src/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Button = ({ label, onClick }) => {
+  return <button onClick={onClick}>{label}</button>;
+};
+
+export default Button;
```

Comment:
```suggestion
export const Button = ({ label, onClick }) => {
  return <button onClick={onClick}>{label}</button>;
};
```

defaulr exportではなくnamed exportするのが一般的です